content/en/metrics/distributions.md

```diff
@@ -14,26 +14,69 @@ further_reading:
 
 Distributions are a metric type that aggregate values sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure.
 
-Global distributions are designed to instrument logical objects, like services, independently from the underlying hosts. Unlike [histograms][1] which aggregate on the Agent-side, global distributions send all raw data collected during the flush interval and the aggregation occurs server-side. Because the underlying data structure has not been aggregated and represents raw data, distributions provide two major features:
+Global distributions instrument logical objects, like services, independently from the underlying hosts. Unlike [histograms][1] which aggregate on the Agent-side, global distributions send all raw data collected during the flush interval and the aggregation occurs server-side using Datadog's [DDSketch data structure][2]. 
+
+Distributions provide enhanced query functionality and configuration options that aren’t offered with other metric types (count, rate, gauge, histogram):
+* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
+  * **Single percentile value over any timeframe**:
+     _“What has the p95 load time for my application been over the past week?”_
+
+  * **Percentile thresholds on metric monitors**:
+    _“Alert me when the p99 of my application’s request latency is greater than 200 ms for the last 5 min.”_
+
+  * **Threshold Queries for SLOs**:
+    _“I’d like to define a 30-day SLO where the p95 of request latency is < 5 seconds”_
```

Comment:
```suggestion
Distributions provide enhanced query functionality and configuration options that aren’t offered with other metric types (count, rate, gauge, histogram):
* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 

  * **Single percentile value over any timeframe**:
  
     _“What has the p95 load time for my application been over the past week?”_

  * **Percentile thresholds on metric monitors**:
  
    _“Alert me when the p99 of my application’s request latency is greater than 200 ms for the last 5 min.”_

  * **Threshold Queries for SLOs**:
  
    _“I’d like to define a 30-day SLO where the p95 of request latency is < 5 seconds”_
```

Reply:
The markdown is being weird in this section, it's displaying the italics underscors and showing it on the same line, so suggest adding some blank lines in there to hopefully fix it.